docker/nginx/conf.d/client.conf

```diff
@@ -440,7 +440,7 @@ server {
 
 		proxy_read_timeout 600;
 		proxy_set_header User-Agent: Sia-Agent;
-		# proxy this call to siad /skynet/skylink/ endpoint (make sure the ip is correct)
```

Comment:
Q: Why remove this comment?
Q: Should we add a new comment so that future readers can understand the purpose for this line? (You can always blame but the commit message is not that helpful).

Reply:
this comment was kind of a leftover from time where this function used ip and we're using alias now which makes the target self explanatory